scipy/stats/_distn_infrastructure.py

```diff
@@ -1407,7 +1407,16 @@ class rv_continuous(rv_generic):
 
       _logpdf, _cdf, _logcdf, _ppf, _rvs, _isf, _sf, _logsf
 
+    The default method ``_rvs`` relies on the inverse of the cdf, ``_ppf``,
+    applied to a uniform random variate. In order to generate random variates
+    efficiently, either the default ``_ppf`` needs to be overwritten (e.g.
+    if the inverse cdf can expressed in an explicit form) or a sampling
+    method needs to be implemented in a custom ``_rvs`` method.
+
     Rarely would you override ``_isf``, ``_sf`` or ``_logsf``, but you could.
+    The main reason would be to improve numerical accuracy: for example,
```

Comment:
This looks fine as it is, but as long as you are here, how about dropping "rarely" from the previous sentence?  I don't think we should say "rarely", given that the default methods are guaranteed to lose precision in the tails.  Plus, of the 111 distributions (continuous and discrete) in scipy, we implement a custom `_sf` for 35 of them, so it is not "rare" in our own implementations.  (I'm also inclined to inspect the behavior of the other 76; they might benefit from implementing `_sf`.)

That sentence could be replaced by something that suggests it is normal, and in many cases even recommended, to override those methods, for the reason that you then explain.